### PR TITLE
Create a core module to limit the dependencies

### PR DESCRIPTION
**Pull request by utix**

### CODE REVIEW

File: core/readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# speedline-core
+
+This is the core module for speedline, use to reduce dependency when it is used as a lib.
+See [speedline](https://github.com/paulirish/speedline) for more details
+
+### Usage
+
+```js
+const speedline = require('speedline');
```
Comment (patrickhulce) [inline core/readme.md]: `speedline-core` :)

File: core/src [hunk 2]
```diff
@@ -0,0 +1 @@
+../src
```
Comment (patrickhulce) [inline core/src:1]: Is this github showing a symlink?

**Review by patrickhulce — COMMENTED** [review]
thanks @utix! 

@paulirish I believe this is the path you preferred, correct?

File: core/src (hunk 2 above)
Comment (utix) [reply core/src:1]: Yep you see it as a link:
Image of Yaktocat
https://github.com/Cdiscount/speedline/tree/master/core

File: core/readme.md (hunk 1 above)
Comment (utix) [reply core/readme.md]: done

**Comment by utix** [comment]
@paulirish anything I can improve to validate this PR ?

**Comment by utix** [comment]
Hi, any feedback to improve this PR ?

**Comment by paulirish** [comment]
thx. finished it in #68

**Comment by paulirish** [comment]
speedline-core@1.4.0
speedline@1.4.1 (the cli)

**Pull request merged**